Index: codeCainUmlEditor/src/main/Java/Main.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>public class Main {\r\n\r\n    public static void main(String[] args) {\r\n        System.out.println(\"hello world\");\r\n    }\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/codeCainUmlEditor/src/main/Java/Main.java b/codeCainUmlEditor/src/main/Java/Main.java
--- a/codeCainUmlEditor/src/main/Java/Main.java	
+++ b/codeCainUmlEditor/src/main/Java/Main.java	
@@ -1,6 +1,8 @@
 public class Main {
 
     public static void main(String[] args) {
-        System.out.println("hello world");
+        System.out.println("Hello world");
     }
+
+
 }
